patches/master_patch.patch

```diff
@@ -2021,34 +1928,55 @@ index 5374bf1652af381a1efc0cdfc96b56b883b716a0..b157bc09a570ca3dd492ffcfb03417e3
  
    bool ShouldSelectReplacement() const {
 diff --git a/third_party/WebKit/Source/core/html/HTMLMediaElement.cpp b/third_party/WebKit/Source/core/html/HTMLMediaElement.cpp
-index 2fc0977882d944317b26a89c095cde076383e308..223eab775d985f06d1775e5141bd4aae6eb5f0d2 100644
+index 4586eb16224502b5ac6f3bfe50d7d5072b708562..ac8b4793be3b631023a16396a35ed7983e0916ba 100644
 --- a/third_party/WebKit/Source/core/html/HTMLMediaElement.cpp
 +++ b/third_party/WebKit/Source/core/html/HTMLMediaElement.cpp
-@@ -2626,7 +2626,7 @@ void HTMLMediaElement::setMuted(bool muted) {
-     }
-   }
+@@ -2495,7 +2495,7 @@ void HTMLMediaElement::setMuted(bool muted) {
+   if (!muted_ && !autoplay_policy_->RequestAutoplayUnmute())
```

Comment:
@hferreiro , why will we need this?
It is already in https://chromium.googlesource.com/chromium/src.git/+/12d76adf4bb340d8e430a18c3a664940157800f6/third_party/WebKit/Source/core/html/HTMLMediaElement.cpp#2458